docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gliderlabs/logspout:v3.2.4 AS build-env
+ENV LOGSPOUT_SUMOLOGIC_VERSION 0.1.0-logspout-3.2.4-dev
```

Comment:
Using the dcd `--version` with this will produce a lot of versions, I think.. like:
```
0.1.0-logspout-3.2.4-dev
0.1.0-logspout-3.2.4
0.1.0-logspout-3.2
0.1.0-logspout-3
0.1.0-logspout
0.1.0
...
```

Running dcd locally with `--dry-run` may help.

Reply:
Maybe run `dcd` with `--tag logspout-3.2.4 ''` then you should get:
```
0.1.0-logspout-3.2.4
0.1-logspout-3.2.4
0.1.0
0.1
logspout-3.2.4
latest
```